MAUI/Barcode-Generator/one-dimensional-symbology.md

```diff
@@ -0,0 +1,450 @@
+    barcode.HeightRequest = 150;
+    barcode.Value = "310194";
+    barcode.Symbology = new UPCE() { Module = 2 };
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator UPC-E Symbology](images/one-dimensional-symbology/maui-upce-symbology.png)
+
+## EAN-13
+
+`EAN-13`, based on the `UPC-A` standard. As like `UPC-A`, it supports only the numeric character. It encodes the 12 digits of input data with the check digit at its end.
+
+* This difference between the `UPCA` and `EAN13` is that number system used in the `EAN13` is 2 digits ranges from 00 to 99 whereas the number system used in `UPCA` is single digits range from 0 to 9.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+    <barcode:SfBarcodeGenerator Value="9735940564824"
+                                ShowText="True"
+                                HeightRequest="150">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:EAN13 Module="2" />
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 150;
+    barcode.Value = "9735940564824";
+    barcode.Symbology = new EAN13() { Module = 2 };
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator EAN-13 Symbology](images/one-dimensional-symbology/maui-ean13-symbology.png)
+
+## EAN-8
+
+`EAN8` is equivalent to the `UPCE` for small packaging details. It is shorter than the `EAN13` barcode and it is longer than `UPCE`.
+
+As like `EAN13` and `UPCE`, it encodes 7 digits of numeric data with the check digit at its end.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+    <barcode:SfBarcodeGenerator Value="11223344"
+                                ShowText="True"
+                                HeightRequest="150">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:EAN8 Module="2" />
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 150;
+    barcode.Value = "11223344";
+    barcode.Symbology = new EAN8() { Module = 2 };
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator EAN-8 Symbology](images/one-dimensional-symbology/maui-ean8-symbology.png)
+
+All the one dimensional symbology supports the `Module` property. The property is used to define the size of the smallest line or dot of the barcode. If this property is not set, the size of the smallest line of the barcode is calculated based on the available size and total number of bars for the provided input value.
```

Comment:
All the one-dimensional symbology supports the `Module` property. This property is used to define the size of the smallest line or dot of the barcode.